lib/crypt_prog.cpp

```diff
@@ -154,7 +121,18 @@ int main(int argc, char** argv) {
         printf("creating keys in %s and %s\n", argv[3], argv[4]);
         n = atoi(argv[2]);
 
-        srand(random_int());
+        retval = random_int(srand_seed);
+        if (retval == 1) {
+            die("can't load ADVAPI32.DLL");
+        } else if (retval == 2) {
+            die("can't open /dev/random");
+        } else if (retval == 3) {
+            die("can't read from /dev/random");
+        } else if (retval) {
+            die("random_int");
+        }
+        srand(srand_seed);
+        
```

Comment:
run locally this command to remove trailing spaces:
`python3 ci_tools/trailing_whitespaces_check.py . --fix`